tgui/packages/tgui/interfaces/NifPanel.js

```diff
@@ -0,0 +1,204 @@
+import { useBackend, useLocalState } from '../backend';
+import { Window } from '../layouts';
+import { Box, Dropdown, LabeledList, ProgressBar, Section, Button, Input, BlockQuote, Flex, Collapsible, Table } from '../components';
+import { TableCell, TableRow } from '../components/Table';
+
+export const NifPanel = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { linked_mob_name, loaded_nifsofts, max_nifsofts, max_power, current_theme } = data;
+  const [settingsOpen, setSettingsOpen] = useLocalState(
+    context,
+    settingsOpen,
+    false
+  );
+
+  return (
+    <Window
+      title={'Nanite Implant Framework'}
+      width={500}
+      height={400}
+      resizable
+      theme={current_theme}>
+      <Window.Content>
+        <Section
+          title={'Welcome to your NIF, ' + linked_mob_name}
```

Comment:
```suggestion
          title={`Welcome to your NIF, ${linked_mob_name}`}
```
Preferred way to format strings in modern JS. I guess you could add a period to the end like
```js
          title={`Welcome to your NIF, ${linked_mob_name}.`}
```
if you feel like it.